### PR TITLE
Sw readme sample fix

### PR DESCRIPTION
**Pull request by shuowu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,12 @@ Install `@okta/okta-react`
 npm install --save @okta/okta-react
 ```
 
-Install peer dependency `@okta/okta-auth-js`
+Install peer dependencies
 
 ```bash
+npm install --save react
+npm install --save react-dom
+npm install --save react-router-dom
```
Comment (mraible) [inline README.md:73]: You don't need to specify `--save` since npm 5.0.0 https://stackoverflow.com/a/19578808/65681. 

You could also shorten this to: `npm install react react-dom react-router-dom`.

**Review by swiftone — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shuowu) [reply README.md:73]: At most times the installation of react, react-dom, and the react-router-dom won't be needed by users. Just list all of the commands separately for easy copy&paste. 
Also, let's keep the `--save` to compatible with older versions of npm.
